.github/workflows/platformDeploy.yml

```diff
@@ -352,6 +352,11 @@ jobs:
         env:
           CF_API_KEY: ${{ secrets.CLOUDFLARE_TOKEN }}
 
+  # Build a version of iOS and Android HybridApp if we are deploying to staging
+  hybridApp:
+    if: ${{ github.event_name == 'release' }}
```

Comment:
This is the event for production deploys, not staging deploys. Staging deploys are currently triggered by pushing tags.

```suggestion
    if: ${{ github.event_name == 'push' }}
```

Reply:
Ah yes - Great catch